### PR TITLE
Watch later

### PR DESCRIPTION
**Pull request by msdeshmukh009**
### Added watch later feature 
watch-later

### CODE REVIEW

File: src/components/sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -37,13 +37,16 @@ const Sidebar = () => {
...
+        >
           <i className="fas fa-clock"></i>
           <span className="text-left">Watch later</span>
         </NavLink>
       </li>
-      <li>
+      {/* <li>
         <NavLink className="grid-30-70" to="/">
           <i className="fas fa-history"></i>
           <span className="text-left">History</span>
```
Comment (Chakravarthi0) [inline src/components/sidebar/Sidebar.jsx:52]: You can remove this comment, if not needed

**Review by Chakravarthi0 — APPROVED** [review]
Nice folder structuring
clean code
good to merge

**Review by manojkumar8400 — APPROVED** [review]
UI looks great good to go.

File: src/components/sidebar/Sidebar.jsx (hunk 1 above)
Comment (msdeshmukh009) [reply src/components/sidebar/Sidebar.jsx:52]: Yes, I will be using it. Thanks for the review.

**Pull request merged**